temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/BalanceRow.kt

```diff
@@ -112,7 +112,7 @@ fun BalanceRow(
             verticalAlignment = Alignment.CenterVertically,
             horizontalArrangement = Arrangement.SpaceBetween
         ) {
-            if (currencyUpfront || doubleRowDisplay) {
+            if (doubleRowDisplay) {
```

Comment:
Can you test and upload screenshots where `doubleRowDisplay` is true and false?

Reply:
If `doubleRowDisplay` is never true we can get rid of it

Reply:
doubleRowDisplay is true for HeaderStickyRow composable.

I will check true and false and upload appropriate screenshots tomorrow.

should I replace line 124 `if (!doubleRowDisplay)` with just an else.